src/plugins/data_source/server/plugin.ts

```diff
@@ -61,16 +61,24 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
     const cryptographyServiceSetup: CryptographyServiceSetup = this.cryptographyService.setup(
       config
     );
+    const dataSourceService: DataSourceServiceSetup = await this.dataSourceService.setup(config);
```

Comment:
Should we name it dataSourceServiceSetUp as cryptographyServiceSetup

Reply:
fixed